README.md

```
@@ -1,337 +1,203 @@
+- getAll() - Listar todos os episódios
+- getById(id) - Buscar episódio por ID
+- search(query) - Buscar por nome
+- getEpisodesByCode(code) - Filtrar por código (S01E01)
+- getMultipleEpisodes(ids) - Buscar múltiplos por IDs
+- getEpisodesWithPagination(page) - Paginação
+
+## Padrões de Design Utilizados
+
+### **1. Repository Pattern**
+dart
+abstract class BaseRepository<T> {
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: T

(MD033, no-inline-html)